tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3636,6 +3636,10 @@ def runTest(self):
         # Prepare TCP packet data
         ttl=64
         pkt_dst_mac=router_mac if router_mac != '' else dst_port_mac
+        is_dualtor = self.test_params.get('is_dualtor', False)
```

Comment:
Is this part of code duplicated with https://github.com/sonic-net/sonic-mgmt/pull/7339/files?